Gemfile.jruby-3.1.lock.release

```diff
@@ -839,7 +839,7 @@ GEM
     unicode-display_width (3.1.4)
       unicode-emoji (~> 4.0, >= 4.0.4)
     unicode-emoji (4.0.4)
-    uri (1.0.3)
+    uri (0.12.3)
```

Comment:
Is this intentional?

Reply:
No, it comes from the hard pin. Need to remove that pin on 9.0 and 9.1 https://github.com/elastic/logstash/blob/0fc31322aa92ad75a04826b30103e9193bf0f006/Gemfile.template#L49C3-L49C82